### PR TITLE
Add mentions about email template modifications in tutorial

### PR DESCRIPTION
**Pull request by SariSaar**
- Add an info box to pricing customization tutorial on updating new line item to email template
- Add an info box to booking unit type tutorial on updating type of line item to email template

### CODE REVIEW

File: src/docs/tutorial-transaction-process/customize-pricing/index.md [hunk 1]
```diff
@@ -654,3 +654,25 @@ included in the price on the booking page. In addition, the Flex Console
 transaction price breakdown also shows the cleaning fee.
 
 ![Cleaning fee in booking breakdown in Flex Console](./consoleBookingBreakdown.png)
+
+<extrainfo title="Bonus: Add cleaning fee to email notifications">
```
Comment (SariSaar) [inline src/docs/tutorial-transaction-process/customize-pricing/index.md:658]:
```suggestion
<extrainfo title="Add cleaning fee to email notifications">
```

File: src/docs/tutorial-transaction-process/create-transaction-process/index.md [hunk 2]
```diff
@@ -432,6 +432,22 @@ _bookingUnitType_ there too:
...
+```html
+{{#eq "line-item/night" code}}
+``` 
+
+with 
+
+```html
+{{#eq "line-item/day" code}}
+```
+in your email notification templates as well to show the correct line items. The next step in the tutorial deals with updating email notifications.
```
Comment (SariSaar) [inline src/docs/tutorial-transaction-process/create-transaction-process/index.md:447]:
```suggestion
in your email notification templates to show the correct line items. The next step in the tutorial deals with updating email notifications.
```

File: src/docs/tutorial-transaction-process/customize-pricing/index.md [hunk 3]
```diff
@@ -654,3 +654,25 @@ included in the price on the booking page. In addition, the Flex Console
 transaction price breakdown also shows the cleaning fee.
 
 ![Cleaning fee in booking breakdown in Flex Console](./consoleBookingBreakdown.png)
+
+<extrainfo title="Add cleaning fee to email notifications">
+If you want to add the cleaning fee to your email notifications, you will need to add it to the email templates.
```
Comment (shareoc) [inline src/docs/tutorial-transaction-process/customize-pricing/index.md:659]:
```suggestion
If you want to add the cleaning fee to your email notifications, you will need to update the email template: money-paid-html.html.
```

Not absolutely necessary, but as this is a tutorial, I think it should be as unambiguous as possible.

Comment (SariSaar) [reply src/docs/tutorial-transaction-process/customize-pricing/index.md:659]: Good point - there are several templates where the change needs to be made, so I'll list them here!

**Pull request merged**